### DISCUSSION TITLE
Is there a dashboard upgrade tool?

### DISCUSSION THREAD

**Discussion by Joey777210** [grafana/grafana#69529] [Dashboards, Panels & Visualizations]
I upgrade my grafana version from 7.* to 9.4, and panels can be viewed fine, but legends color change are disabled... 
Compared dashboard JSON Model between 7 and 9, there are many difference. 
So, Is there a dashboard upgrade tool? To auto justify JSON Model so everything will be ok after version upgrade. 
If not, I can make one if needed.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
